client/state/site-settings/exporter/actions.js

```diff
@@ -2,16 +2,33 @@
  * Internal dependencies
  */
 import notices from 'notices';
+import wpcom from 'lib/wp';
+import debugModule from 'debug';
 import i18n from 'lib/mixins/i18n';
 
+import { prepareExportRequest, getDataState } from './selectors';
+import { getSelectedSite } from 'state/ui/selectors';
+
+const debug = debugModule( 'calypso:exporter' );
+const wpcomUndocumented = wpcom.undocumented();
+
 import {
 	SET_EXPORT_POST_TYPE,
+	SET_EXPORTER_ADVANCED_SETTING,
+
+	FETCH_EXPORTER_ADVANCED_SETTINGS,
+	RECEIVE_EXPORTER_ADVANCED_SETTINGS,
+
 	REQUEST_START_EXPORT,
 	REPLY_START_EXPORT,
 	FAIL_EXPORT,
 	COMPLETE_EXPORT
 } from 'state/action-types';
 
+const ERROR_START_EXPORT = i18n.translate( 'We had a problem starting your export. Please check your internet connection and try again.' );
+const ERROR_POLLING = i18n.translate( 'We had a problem checking the status of your export. Please check your internet connection and try again.' );
+const ERROR_PROCESSING = i18n.translate( 'We had a problem processing your export. Please contact support.' );
```

Comment:
A 'Get Help' button appears on the notice - that's defined [on line 203](https://github.com/Automattic/wp-calypso/pull/1359/files#diff-81200b59520567c2f9f7622d4d721981R203). I've changed the link to /help/contact.